src/Microsoft.Azure.WebJobs.Host.Storage/WebJobs.Host.Storage.csproj

```diff
@@ -6,6 +6,7 @@
   <Import Project="..\..\build\common.props" />  
   
   <PropertyGroup>
+    <Version>4.0.0$(VersionSuffix)</Version>
```

Comment:
Can you elevate this up to common.props? Make a `HostStorageVersion` property and then use it here? It makes it easier whenever we need to bump versions that they're all in the same file.

Reply:
I'll make a follow-up change to adjust the versions to the main file here.